types/node/v18/crypto.d.ts

```diff
@@ -994,7 +1026,20 @@ declare module "crypto" {
     /** @deprecated since v10.0.0 use `createDecipheriv()` */
     function createDecipher(algorithm: CipherGCMTypes, password: BinaryLike, options?: CipherGCMOptions): DecipherGCM;
     /** @deprecated since v10.0.0 use `createDecipheriv()` */
-    function createDecipher(algorithm: string, password: BinaryLike, options?: stream.TransformOptions): Decipher;
+    function createDecipher(algorithm: CipherOCBTypes, password: BinaryLike, options: CipherOCBOptions): DecipherOCB;
+    /** @deprecated since v10.0.0 use `createDecipheriv()` */
+    function createDecipher(
+        algorithm: CipherChaCha20Poly1305Types,
+        password: BinaryLike,
+        options?: CipherChaCha20Poly1305Options,
+    ): DecipherChaCha20Poly1305;
+    /** @deprecated since v10.0.0 use `createDecipheriv()` */
+    // eslint-disable-next-line @definitelytyped/no-unnecessary-generics
+    function createDecipher<T extends string>(
```

Comment:
Why is adding this signature with a type parameter necessary?

Reply:
Fixed.